package.json

```diff
@@ -75,37 +74,36 @@
     "@storybook/addons": "^5.3.19",
     "@storybook/react": "^5.3.19",
     "@storybook/storybook-deployer": "^2.8.6",
-    "@testing-library/react": "^8.0.8",
+    "@testing-library/react": "^10.4.8",
     "abstract-sdk": "^8.0.0-beta.33",
     "auto": "^9.50.1",
-    "babel-jest": "^26.1.0",
+    "babel-jest": "^26.3.0",
     "babel-loader": "^8.1.0",
     "babel-plugin-module-name-mapper": "^1.2.0",
     "babel-plugin-module-resolver": "^4.0.0",
     "css-loader": "^2.1.1",
-    "eslint": "^7.5.0",
+    "eslint": "^7.6.0",
     "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-jest-enzyme": "^6.0.2",
     "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-babel-module": "^5.1.2",
     "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest": "^23.18.0",
+    "eslint-plugin-jest": "^23.20.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-prettier": "^3.0.1",
-    "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-react-hooks": "^1.7.0",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^4.0.8",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.1.0",
-    "jest-dom": "^3.2.2",
-    "loki": "^0.20.3",
+    "jest": "^26.3.0",
+    "jest-dom": "^4.0.0",
+    "loki": "^0.24.0",
```

Comment:
This is a sizable bump in Loki. Did you check that VRT still successfully runs for you locally?

Reply:
I didn't tell Loki to update which is strange. I'll check it out.